src/api/cvc4cpp.cpp

```diff
@@ -49,6 +49,7 @@
 #include "expr/node_manager.h"
 #include "expr/sequence.h"
 #include "expr/type.h"
```

Comment:
Can we remove this? Or not yet?

Reply:
Not yet, there is a annoying case where we need to use ExprManager mkVar + flags in the parser, which requires passing a type.